cypress/integration/score_card_spec.js

```diff
@@ -2,7 +2,7 @@ describe('Scorecard user flow', () => {
 
   beforeEach(() => {
     cy.visit('http://localhost:3000/')
-      .get('.search-btn').click().click()
+      .get('.search-btn').click()
```

Comment:
Great find to fix our bug and adjusting our click just to fire once and not on double click for our search button!